contributors/guide/pull-requests.md

```diff
@@ -590,7 +597,32 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
-## Fixing linter issues
+## Large or Automatic Edits
+
+Some tools make it very easy to create large Pull Requests, for example:
+- global search/replace
+- linters which automatically correct issues (see also next section)
+- large language models (LLMs) which generate code or documentation
+
+To make it easier for reviewers to handle such Pull Requests, please explain
+how it was generated in the "Special notes for your reviewer" section of the
+Pull Request description. Reviewers may then be able to reproduce those steps
+(search/replace, linters) or can start the review with the right expectations
+(LLMs). Also consider the section about [splitting up Pull
+Requests](#dont-open-pull-requests-that-span-the-whole-repository) above.
+
+Even with such tools it is still your responsibility as submitter of a Pull
+Request to ensure that the change is correct (to the best of your knowledge),
+and that making the change improves the project enough to justify the cost that
+is needed to review and merge the Pull Request (see previous section). If
+unsure, create a draft Pull Request and ask for guidance.
```

Comment:
```suggestion
unsure, create a Draft Pull Request and ask for guidance.
```
as mentioned elsewhere, this will be consistent with github UI, ie. we won't have to explain it :wink: